This patch was prepared by Gwenole Beauchesne <gbeauchesne@mandrakesoft.com> for Mandrake
Add SAL_FONTPATH_USER to the font path
--- oo_1.0_src/sysui/oounix/office/scripts/soffice.sh.user-fontpath	2002-04-01 21:56:18.000000000 +0200
+++ oo_1.0_src/sysui/oounix/office/scripts/soffice.sh	2002-05-03 17:23:46.000000000 +0200
@@ -179,7 +179,7 @@ THREADS_TYPE=native_threads
 
 # misc. environment variables
 SAL_FONTPATH="$sd_fonts/75dpi:unscaled;$sd_fonts/75dpi"
-SAL_FONTPATH_PRIVATE="$sd_fonts/truetype;$sd_fonts/type1;$sd_fonts/serverfonts;$userinst/user/fonts"
+SAL_FONTPATH_PRIVATE="$sd_fonts/truetype;$sd_fonts/type1;$sd_fonts/serverfonts;$userinst/user/fonts;$SAL_FONTPATH_USER"
 XPPATH="$sd_inst/share/xp3"
 MOZILLA_FIVE_HOME="$sd_inst/program"
 
